File: src/main/java/com/viaversion/aas/command/VersionCommand.java
package com.viaversion.aas.command;

import com.viaversion.viaversion.api.command.ViaCommandSender;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public enum VersionCommand implements Command {
	INSTANCE;

	@NotNull
	@Override
	public String getInfo() {
		return "Alias of 'viaver viaaas'";
	}

	@NotNull
	@Override
	public List<String> suggest(@NotNull ViaCommandSender sender, @NotNull String alias, @NotNull List<String> args) {
		return List.of();
	}

	@Override
	public void execute(@NotNull ViaCommandSender sender, @NotNull String alias, @NotNull List<String> args) {
		ViaAspirinCommand.INSTANCE.execute(sender, alias, List.of("viaaas"));
	}
}

File: src/main/kotlin/com/viaversion/aas/command/VersionCommand.kt
package com.viaversion.aas.command

import com.viaversion.aas.AspirinServer
import com.viaversion.viaversion.api.command.ViaCommandSender
import kotlinx.coroutines.CoroutineScope
import kotlinx.coroutines.Job
import kotlinx.coroutines.launch

class VersionCommand : Command {
    override val info: String
        get() = "Shows VIAaaS version and checks for updates"

    override fun suggest(sender: ViaCommandSender, alias: String, args: List<String>): List<String> {
        return listOf()
    }

    override fun execute(sender: ViaCommandSender, alias: String, args: List<String>) {
        sender.sendMessage("VIAaaS version ${AspirinServer.version}")
        CoroutineScope(Job()).launch { sender.sendMessage(AspirinServer.updaterCheckMessage()) }
    }
}